provider/provider.go

```diff
@@ -38,7 +38,9 @@ func New() *schema.Provider {
 			"feature_use_managed_variables": {
 				Type:        schema.TypeBool,
 				Description: "Feature: use managed Terraform variables. If disabled, Terraform variables will be included in legacy Parameter Schema.",
+				Default:     true,
 				Optional:    true,
+				Deprecated:  "Terraform variables are now exclusively utilized for template-wide variables after the removal of support for legacy parameters.",
```

Comment:
This is deprecated, but still present. Should the description say something like: setting this is a no-op? Is that true or what would it mean for a template author to have this set to false?

Reply:
Maybe we should add a sentence clarifying that this property is not used anymore?